src/vs-bicep/Bicep.VSLanguageServerClient/BicepLanguageServerClient.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Bicep.VSLanguageServerClient.ContentType;
+using Bicep.VSLanguageServerClient.MiddleLayerProviders;
+using Bicep.VSLanguageServerClient.ProcessLauncher;
+using Bicep.VSLanguageServerClient.ProcessTracker;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.Setup.Configuration;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Threading;
+using Microsoft.VisualStudio.Utilities;
+using StreamJsonRpc;
+
+namespace Bicep.VSLanguageServerClient
+{
+    [Export(typeof(ILanguageClient))]
+    [ContentType(BicepContentTypeDefinition.ContentType)]
+    public class BicepLanguageServerClient : ILanguageClient, ILanguageClientCustomMessage2
+    {
+        private IClientProcess? _process;
```

Comment:
For sake of consistency, should we stick to the field naming convention to not use the `_` prefix?

Reply:
Updated this and few other files I added:
https://github.com/Azure/bicep/commit/afee5cdf4c813f69563636747d75af098cb94fd9